Sources/XCBProtocol_13_0/Indexing Info/IndexingInfoRequest.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import MessagePack
+import XCBProtocol
+
+public struct IndexingInfoRequest: Decodable {
+    public let sessionHandle: String
+    public let responseChannel: UInt64
+    public let buildRequest: BuildRequest // Called `request` by Xcode
```

Comment:
These `// Called `XYZ` by Xcode` like comments aren't needed when we have `CodingKeys`.